File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Three/C++/prog4.cpp
#include<iostream>

int main() {
	int num;
	for(int i = 0; i < 4; i++) {
		num = 7 - i;
		for(int j = 0; j <= i; j++) {
			std::cout << num << " ";	
			num--;
		}
		std::cout << "\n";
	}
	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week9/19Mar/51-DailyFlash_Solutions/19-Mar/Java/prog4.java
class Demo {
	public static void main(String[] args) {
		int num = 1;
		for(int i = 0; i < 4; i++) {
			for(int j = 0; j < 7 - i; j++) {
				if(j < i) {
					System.out.print(" ");
				} else {
					if(i % 2 == 0) {
						System.out.print(num);
						num++;
						if(j == 6 - i)
							num--;
					}

					else {
						System.out.print(num);
						num--;
						if(j == 6 - i)
							num++;
					}
				}
			}
			System.out.println();
		}
	}
}
